pkg/kubelet/dockertools/manager.go

```diff
@@ -216,12 +218,30 @@ func NewDockerManager(
 		cpuCFSQuota:            cpuCFSQuota,
 	}
 	dm.runner = lifecycle.NewHandlerRunner(httpClient, dm, dm)
-	if serializeImagePulls {
-		dm.imagePuller = kubecontainer.NewSerializedImagePuller(kubecontainer.FilterEventRecorder(recorder), dm, imageBackOff)
+	dm.containerGC = NewContainerGC(client, containerLogsDir)
+
+	minimumDockerAPIVersionForParallelPullsFlag := false
```

Comment:
Same request as before - please create a helper func with a name something like `shouldSerializePulls`.

Also, the `bool` name should be something like `dockerSupportsParallelPulls`